docker/wait_for_pg

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+# Usage: wait_for_pg
+# expected ENV: POSTGRES_HOST, POSTGRES_PORT, POSTGRES_USER, POSTGRES_PASSWORD
+# Description:
+# Waits for postgres to be fully up and running
+
+export PGPASSWORD=${POSTGRES_PASSWORD-postgres}
+
+PROBE_TIMEOUT=${PROBE_TIMEOUT-15}
+POSTGRES_HOST=${POSTGRES_HOST-postgres}
+POSTGRES_PORT=${POSTGRES_PORT-5432}
+POSTGRES_USER=${POSTGRES_USER-postgres}
+
+log() {
+  echo "[wait-for] [`date +'%Y%m%d%H%M%S'`] $@"
+}
+
+wait_for() {
```

Comment:
This function seems meaty enough that we'd want a test.

Reply:
🙋‍♂️ https://pkgs.alpinelinux.org/package/edge/testing/x86_64/dockerize is something that has this functionality out of the box if required.